git-tree.py

```diff
@@ -200,7 +201,7 @@ def calculate_branch_column_width(print_outs, branches):
     return max(branch_column_widths) + 2
 
 
-def print_table(print_outs, branches):
+def print_table(print_outs, branches, highlight_branch=""):
```

Comment:
Nit: may be good to provide docstrings so it's easy to quickly see what `highlight_branch` is used for.